src/content/docs/developer-tools/sdks/backend/nodejs-sdk.mdx

```diff
@@ -16,7 +16,7 @@ You can view the [NodeJS docs](https://github.com/kinde-oss/kinde-nodejs-sdk) a
 
 ## Requirements
 
-- Node version 18.x.x or newer.
+- Node version 18.x.x or newer. If used in conjunction with the ExpressJS sdk [Express SDK](https://github.com/kinde-oss/kinde-node-express), then Node 20.x.x or new is required.
```

Comment:
@brettchaldecott - shouldnt there be changes to the [Express SDK doc](https://github.com/kinde-oss/documentation/blob/main/src/content/docs/developer-tools/sdks/backend/express-sdk.mdx), not the NodeJS SDK doc?